src/eval.ml

```diff
@@ -138,6 +145,14 @@ and eq_modulo : term -> term -> bool = fun a b ->
     | (Appl(t1,u1), Appl(t2,u2)) -> eq_modulo ((u1,u2)::(t1,t2)::l)
     | (Meta(m1,a1), Meta(m2,a2)) when m1 == m2 ->
         eq_modulo (if a1 == a2 then l else List.add_array2 a1 a2 l)
+    (* Try a Δ conversion *)
```

Comment:
δ

Reply:
Capital for local bindings <https://coq.inria.fr/distrib/current/refman/language/cic.html#delta-reduction>

Reply:
Perhaps but this is not common terminology. Please use δ.